ios/Capacitor/Capacitor/CAPBridgedPlugin.h

```diff
@@ -16,16 +16,14 @@
 @class CAPPlugin;
 
 @protocol CAPBridgedPlugin <NSObject>
-+(NSString *)pluginId;
-+(NSString *)jsName;
-+(NSArray *)pluginMethods;
-+(CAPPluginMethod *)getMethod:(NSString *)methodName;
-@optional
+@property (nonnull, readonly) NSString *identifier;
+@property (nonnull, readonly) NSString *jsName;
+@property (nonnull, readonly) NSArray<CAPPluginMethod *> *pluginMethods;
```

Comment:
Well this is a much needed update. That's some old school ObjC. The non-null will help a lot with swift. 💯